### ISSUE TITLE
YieldWallet-Gautam#4458: Empty epoch issue with Block producer node

### ISSUE THREAD

**Issue by gsampathkumar** [MinaProtocol/mina#6075]
Having issues with Block producer (Snark worker is fine). Block producer complains about 'empty epoch' and stops after a few minutes. Log below:

2020-09-21 13:51:06 UTC [Info] why u fail to validate :(
2020-09-21 13:51:12 UTC [Info] Will start initialization without connecting with too many peers
2020-09-21 13:51:12 UTC [Info] Requesting peers for their best tip to do initialization
2020-09-21 13:51:12 UTC [Info] Starting Transition Frontier Controller phase
2020-09-21 13:51:12 UTC [Info] Coda daemon is now synced
2020-09-21 13:51:12 UTC [Info] Updating new available work took 0.03147125244140625 ms
2020-09-21 13:51:12 UTC [Info] Determining next slot to produce block
2020-09-21 13:51:12 UTC [Fatal] An empty epoch is detected! This could be caused by the following reasons: system time is out of sync with protocol state time; or internet connection is down or unstable; or the testnet has crashed. If it is the first case, please setup NTP. If it is the second case, please check the internet connection. If it is the last case, in our current version of testnet this is unrecoverable, but we will fix it in future versions once the planned change to consensus is finished.

**Comment by garethtdavies** [comment]
This is a duplicate of #6094

**Comment by shimkiv** [comment]
Closing this issue as it is `stale` for a long time.
